boards/nrf52840dongle/doc.txt

```diff
@@ -0,0 +1,58 @@
+/**
+@defgroup    boards_nrf52840dongle nRF52840-Dongle
+@ingroup     boards
+@brief       Support for the nRF52840-Dongle
+
+### General information
+
+The nRF52840-Dongle is a bare USB-stick shaped device that houses barely
+anything than the nRF52840 itself, which offers BLE and 802.15.4 and USB
+connectivity.
+
+Unlike similar sticks (like the @ref boards_nrf52840-mdk), it features no
+dedicated programmer hardware but relies on direct USB communication with a
+built-in bootloader.
+
+The board features two LEDs (LD1: green, LD2: RGB), a user (SW1) and a
+reset button as well as 15 configurable external pins.
+
+### Links
+
+- [nRF52840 web page](https://www.nordicsemi.com/?sc_itemid=%7BCDCCA013-FE4C-4655-B20C-1557AB6568C9%7D)
+- [documentation and hardware description](https://infocenter.nordicsemi.com/topic/ug_nrf52840_dongle/UG/nrf52840_Dongle/intro.html?cp=3_0_5)
+
+### Flash the board
+
+The board is flashed using its on-board boot loader; the
+[nrfutil](https://github.com/NordicSemiconductor/pc-nrfutil) program needs to
+be installed. That can turn the binary into a suitable zip file and send it to the
+bootloader.
+
+The process is automated in the usual `make flash` target. For that process to
+complete, you need to enter the bootloader manually by pressing the board's
+reset button. Readiness of the bootloader is indicated by LD2 pulsing in red.
```

Comment:
I think if you enable the `bootloader_nrfutil` feature the bootloader reset will be triggered automatically without having to press any buttons.

Reply:
(That of course will only work if RIOT didn’t crash, as the reset is triggered by the USB stack)

Reply:
That would only be the case if FEATURES_REQUIRED += bootloader_nrfutil / USEMODULE += usb_board_reset were set like in adafruit-clue (along with the usb_board_reset_in_bootloader from there), and I'd prefer not to introduce that hack onto platforms where it's not the default convention of the hardware.

Reply:
> I'd prefer not to introduce that hack onto platforms where it's not the default convention of the hardware.

The board is not provided by default with the nrfutil bootloader ?

Reply:
Well there is the added convenience of not having to press any buttons when doing `make flash` (and the added convenience that this will probably solve the CI problem too)